### PR TITLE
*: support showing forwarded status in Grafana

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve?

Problem Summary: This PR is going to show the forwarded status in Grafana

### What is changed and how it works?

What's Changed: Add a new panel in PD client

How it Works:
Screen Shot 2021-03-29 at 8 06 36 PM

### Check List 

Tests 

- No code

### Release note 

- support showing the forwarded status of PD client in Grafana

### CODE REVIEW

File: metrics/grafana/tidb.json [hunk 1]
```diff
@@ -8013,7 +8013,7 @@
             "h": 7,
             "w": 8,
             "x": 16,
-            "y": 17
+            "y": 16
```
Comment (BusyJay) [inline metrics/grafana/tidb.json:8016]: Is it expected change?

Comment (rleungx) [reply metrics/grafana/tidb.json:8016]: I think it doesn't matter.

Comment (BusyJay) [reply metrics/grafana/tidb.json:8016]: Why? It can change the position of the graph.

File: metrics/grafana/tidb.json [hunk 2]
```diff
@@ -802,7 +803,6 @@
           },
           "yaxes": [
             {
-              "$$hashKey": "object:518",
```
Comment (BusyJay) [inline metrics/grafana/tidb.json:805]: Why change this?

Comment (rleungx) [reply metrics/grafana/tidb.json:805]: Not sure, I just import, add new panel, and export.

**Review by BusyJay — APPROVED** [review]

**Review by nolouch — APPROVED** [review]

**Pull request merged**